tiledb/sm/rest/rest_profile.cc

```diff
@@ -184,17 +167,9 @@ const std::string* RestProfile::get_param(const std::string& param) const {
  * See issue [#727](https://github.com/nlohmann/json/issues/727) for details.
  */
 void RestProfile::save_to_file(const bool overwrite) {
-  // Validate that the profile is complete (if username is set, so is password)
```

Comment:
I'd suggest keeping those restrictions to help the user set up authentication correctly.

Reply:
ok, added them back.